commit 34a4831835d8b221f4afb8ec29b511c133228791
Author: Anthony Xu <anthony.xu@intel.com>
Date:   Mon Mar 14 15:08:31 2016 -0700

    disable kvmvapic, most of servers have flexpriority VT technology,
    kvmvapic is not needed.
    
    Signed-off-by: Anthony Xu <anthony.xu@intel.com>

diff --git a/hw/intc/apic_common.c b/hw/intc/apic_common.c
index 659f377..bb726a7 100644
--- a/hw/intc/apic_common.c
+++ b/hw/intc/apic_common.c
@@ -413,8 +413,10 @@ static const VMStateDescription vmstate_apic_common = {
 static Property apic_properties_common[] = {
     DEFINE_PROP_UINT8("id", APICCommonState, id, -1),
     DEFINE_PROP_UINT8("version", APICCommonState, version, 0x14),
+#if 0 //disable kvmvapic
     DEFINE_PROP_BIT("vapic", APICCommonState, vapic_control, VAPIC_ENABLE_BIT,
                     true),
+#endif
     DEFINE_PROP_END_OF_LIST(),
 };
 
